Pod/Classes/Lifecycle/Behaviors/Nav-Bar-Hairline-Fade/NavBarHairlineFadeBehavior.swift

```diff
@@ -45,13 +45,20 @@ public extension NavBarHairlineFadeBehavior {
         set { navBarHairlineFadeUpdater.hairline.thickness = newValue }
         get { return navBarHairlineFadeUpdater.hairline.thickness }
     }
+    
+    var logsContentOffset: Bool {
+        set { navBarHairlineFadeUpdater.logsContentOffset = newValue }
+        get { return navBarHairlineFadeUpdater.logsContentOffset}
```

Comment:
#req whitespace
```suggestion
        get { return navBarHairlineFadeUpdater.logsContentOffset }
```

Reply:
also #nth we can omit the return value if we can guarantee that everyone is using Swift 5.1, except I don't think we can guarantee that, so never mind 😞